docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -328,7 +460,51 @@ Or to set the column ``my_new_partition_column`` as a partition column on a tabl
 
 The current values of a table's properties can be shown using :doc:`SHOW CREATE TABLE </sql/show-create-table>`.
 
-.. _iceberg-type-mapping:
+COMMENT
+~~~~~~~
+
+The Iceberg connector supports setting comments on the following objects:
```

Comment:
could also just do 

on tables, views, and columns with the COMMENT statement.

but .. also ALTER x ADD COMMENT and such works too right?

Reply:
`ALTER x ADD COMMENT` I'm not aware of this syntax. I haven't seen also any occurrence of "ADD COMMENT" in the codebase.

Reply:
Sorry for not being clear .. I meant this stuff for example 

ALTER TABLE [ IF EXISTS ] name ADD COLUMN [ IF NOT EXISTS ] column_name data_type
  [ NOT NULL ] [ COMMENT comment ]

But also setting comments when using create 

CREATE TABLE [ IF NOT EXISTS ]
table_name (
  { column_name data_type [ NOT NULL ]
      [ COMMENT comment ]
      [ WITH ( property_name = expression [, ...] ) ]
  | LIKE existing_table_name
      [ { INCLUDING | EXCLUDING } PROPERTIES ]
  }
  [, ...]
)
[ COMMENT table_comment ]


In that case .. table and column comments..

Reply:
We should verify and document that they all work for tables and views.. also what about materialized views?

Reply:
I added corresponding mentions for both `CREATE TABLE` , `ALTER TABLE`. Thanks for the suggestion.

Setting comments on MV is not supported.